music21/stream/base.py

```diff
@@ -2908,6 +2908,14 @@ def splitAtDurations(self, *, recurse=False) -> base._SplitTuple:
         >>> result = m.splitAtDurations(recurse=True)
         >>> list(result[0][note.Rest])
         [<music21.note.Rest whole>, <music21.note.Rest quarter>]
+
+        >>> r2 = note.Rest(quarterLength=5.0)
```

Comment:
Can we add a note explaining that 'fullMeasure rests are not split at durations, since they will be expressed as a single whole note in most scores anyhow'